### PR TITLE
HTTPS support

### PR DESCRIPTION
**Pull request by eugene-manuilov**
### Description of the Change

Added one more question to the `create` command that enables HTTPS connection for a new site. If HTTPS is selected, then Nginx config file will be altered to listen to 443 port and use pre-generated certificate which is also added to this PR. The certificate is self-signed with `-days 9999` argument to make sure we don't need to regenerate it next dozens of years.

### Benefits

Engineers will be able to create local sites with HTTPS connections.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

The original request came from: https://github.com/10up/wp-local-docker-docs/issues/19

### CODE REVIEW

**Comment by dustinrue** [comment]
SSL support exists in the project already and is implemented in the global nginx proxy. However, setup is semi manual and requires a search-replace after site creation to setup properly. I like the addition of a question and having WordPress installed using https...would you be willing to refactor this so that it asks the question about enabling SSL and if yes then to install WordPress as https://<domain> ?

File: src/wordpress.js [hunk 1]
```diff
@@ -48,7 +50,7 @@ const install = async function( env, envHost, answers ) {
             break;
     }
 
-    execSync( `docker-compose exec phpfpm wp core ${command} ${flags} --url=http://${envHost} --title="${answers.title}" --admin_user="${answers.username}" --admin_password="${answers.password}" --admin_email="${answers.email}"`, { stdio: 'inherit', cwd: envPath });
+    execSync( `docker-compose exec phpfpm wp core ${command} ${flags} --url=${http}://${envHost} --title="${answers.title}" --admin_user="${answers.username}" --admin_password="${answers.password}" --admin_email="${answers.email}"`, { stdio: 'inherit', cwd: envPath });
```
Comment (eugene-manuilov) [inline src/wordpress.js]: @dustinrue it already does it. If SSL is enabled, then WordPress is installed as https://. See it here

Comment (dustinrue) [reply src/wordpress.js]: Right, but I'm saying everything else in the PR is not necessary as an SSL cert is shipped with wp-local-docker and is on at all times.

**Comment by eugene-manuilov** [comment]
Thanks @dustinrue. I have updated my PR. Could you please check it again?

**Comment by cinghaman** [comment]
Is it merged to the latest 2.7.0?

**Pull request merged**